### PR TITLE
feat: add new bootstrap nodes; sort order for dialing discovered ma's

### PR DESCRIPTION
**Pull request by d-roak**
props to @sfroment for the original sorting function for multiaddrs

### CODE REVIEW

File: packages/network/src/node.ts [hunk 1]
```diff
@@ -149,9 +150,25 @@ export class TopologyNetworkNode {
...
+				const localRegex =
+					/(^127\.)|(^10\.)|(^172\.1[6-9]\.)|(^172\.2[0-9]\.)|(^172\.3[0-1]\.)|(^192\.168\.)/;
+				const aLocal = localRegex.test(a.toString());
+				const bLocal = localRegex.test(b.toString());
+				const aWebrtc = a.toString().includes("/webrtc/");
+				const bWebrtc = b.toString().includes("/webrtc/");
+				if (aLocal && !bLocal) return 1;
+				if (!aLocal && bLocal) return -1;
+				if (aWebrtc && !bWebrtc) return -1;
+				if (!aWebrtc && bWebrtc) return 1;
```
Comment (trungnotchung) [inline packages/network/src/node.ts:163]: to ensure WebRTC addresses are dialed before non-local ones, i think we should update the condition order

File: packages/network/src/node.ts [hunk 2]
```diff
@@ -149,9 +150,25 @@ export class TopologyNetworkNode {
...
+				return 0;
+			});
+
+			// Dial WebRTC addresses first, then non-local addresses
+			for (const ma of sortedAddrs) {
```
Comment (trungnotchung) [inline packages/network/src/node.ts:168]: consider renaming `ma` to `address` for clarity

File: packages/network/src/node.ts (hunk 1 above)
Comment (d-roak) [reply packages/network/src/node.ts:163]: ordering imo should be:
- all non-local
  - all webrtc
  - all non-webrtc
- all local
  - all webrtc
  - all non-webrtc

wdyt?
also, is the logic compliant with that ordering?

File: packages/network/src/node.ts (hunk 2 above)
Comment (d-roak) [reply packages/network/src/node.ts:168]: do you know how to create suggestions while reviewing? for this types of comment u can do straight a suggestion
image

File: packages/network/src/node.ts (hunk 1 above)
Comment (trungnotchung) [reply packages/network/src/node.ts:163]: `Dial WebRTC addresses first, then non-local addresses`: this comment in line 167 makes me think that the order looks like:
* all webrtc 
 * all non-local
 * all local
* all non-webrtc
 * all non-local
 * all local

Comment (d-roak) [reply packages/network/src/node.ts:163]: let's change the comment then

Comment (elielnfinic) [reply packages/network/src/node.ts:163]: When testing, this is the order I am getting : 
- webrtc
   - local
   - non-local 
- non-webrtc
  - local
  - non-local

E.g 
Input :  ```[
    "/ip4/127.0.0.1/tcp/4001/p2p/12D3KooWQJ",
    "/ip4/127.0.0.1/tcp/4001/webrtc/p2p/12D3KooWQJ",
    "/ip4/192.168.0.1/tcp/4001/webrtc/p2p/12D3KooWQJ",
    "/ip4/172.20.34.1/tcp/4001/webrtc/p2p/12D3KooWQJ",
    "/ip4/20.34.1/tcp/4001",
    "/ip4/20.34.1/tcp/4001/webrtc/p2p/12D3KooWQJ",
]```

Output : ```[
  '/ip4/127.0.0.1/tcp/4001/webrtc/p2p/12D3KooWQJ',
  '/ip4/192.168.0.1/tcp/4001/webrtc/p2p/12D3KooWQJ',
  '/ip4/172.20.34.1/tcp/4001/webrtc/p2p/12D3KooWQJ',
  '/ip4/20.34.1/tcp/4001/webrtc/p2p/12D3KooWQJ',
  '/ip4/127.0.0.1/tcp/4001/p2p/12D3KooWQJ',
  '/ip4/20.34.1/tcp/4001'
]```

**Review by elielnfinic — CHANGES_REQUESTED** [review]

File: packages/network/src/node.ts (hunk 1 above)
Comment (d-roak) [reply packages/network/src/node.ts:163]: good catch, the regex is wrong

**Review by elielnfinic — APPROVED** [review]
Sort is working perfectly.

**Review by trungnotchung — APPROVED** [review]

**Pull request merged**